core/services/fluxmonitor/flux_monitor.go

```diff
@@ -661,15 +661,28 @@ func (p *PollingDeviationChecker) SetOracleAddress() error {
 			}
 		}
 	}
+
+	loggerFields := []interface{}{
+		"jobID", p.initr.JobSpecID.String(),
+		"contract", p.initr.Address.Hex(),
+		"accounts", accounts,
+		"oracleAddresses", oracleAddrs,
+	}
+
 	if len(accounts) > 0 {
 		addr := accounts[0].Address
-		logger.Warnw("None of the node's keys matched any oracle addresses, using first available key. This flux monitor job may not work correctly", "address", addr, "accounts", accounts, "oracleAddresses", oracleAddrs)
+
+		loggerFields = append(loggerFields, "address", addr)
```

Comment:
You might need to manually `.Hex` it otherwise it could come out as base64 or something wild

Reply:
Fixed